pages/components/Collection/Collection.js

```diff
@@ -1,5 +1,94 @@
-const Collection = () => (
-  <h1>123</h1>
-)
+import styles from "@/pages/components/Collection/Collection.module.scss"
```

Comment:
A typical way to export styles as `S`. It makes the code shorter
`import styles as S from "@/pages/components/Collection/Collection.module.scss"`

`S.info__about_bold` instead `styles.info__about_bold`

Reply:
If it is a modular CSS and CSS classes will be hashed, like `title-h343sgh`, then there is no reason to use `BEM` at all. All classes will be scoped and hashed in the module and BEM does not have any value for us then.
 
So we have `S.about_bold` instead of `styles.info__about_bold`.

Reply:
Thanks, fixed